script.js

```diff
@@ -1,4 +1,160 @@
+// Global variables
+var currentMode = "start";
+var reversedMode = false; // adding reversed mode
+var myOutputValue = "";
+var player = "";
+var player1Score = 0;
+var player2Score = 0;
+var player1DiceOrder = [];
+var player2DiceOrder = [];
+var player1FinalNumb = 0;
+var player2FinalNumb = 0;
+
+// Generate dices
+var diceRollGenerator = function () {
+  return Math.floor(Math.random() * 6) + 1;
+};
+// Specifying the 2 dices for each player
+var diceRoll1Player1 = diceRollGenerator();
+var diceRoll2Player1 = diceRollGenerator();
+var diceRoll1Player2 = diceRollGenerator();
+var diceRoll2Player2 = diceRollGenerator();
+
+// Rolling the dice for player 1
+var player1RollDice = function () {
+  return `Player 1: Dice 1 rolled ${diceRoll1Player1} and Dice 2 rolled ${diceRoll2Player1}. <br><br> Please input 1 or 2 to choose the order of your dice roll.`;
+};
+
+// Order of the dices for player 1
+var userDiceOrderPlayer1 = [];
+var diceOrder1Player1 = function () {
+  userDiceOrderPlayer1.push(diceRoll1Player1, diceRoll2Player1);
+  return `Player 1: Your number is ${(player1FinalNumb = userDiceOrderPlayer1
+    .join("")
+    .trim())}.<br><br> Player 2 press Enter to roll your dice.`; // to combine the 0 and 1 index together
+};
+var diceOrder2Player1 = function () {
+  userDiceOrderPlayer1.push(diceRoll2Player1, diceRoll1Player1);
+  return `Player 1: Your number is ${(player1FinalNumb = userDiceOrderPlayer1
+    .join("")
+    .trim())}. <br><br> Player 2 press Enter to roll your dice.`; // to combine the 0 and 1 index together
+};
```

Comment:
These 2 functions together with the ones in lines 49 to 62 can potentially be combined into 1 function.
Where a conditional determines the order the dice are concatenated in. 

And with regards to P1, P2 scores, you could consider a global variable var player1Score, var player2Score, where you store their score at the end of the function after concatenating them.